lib/sles4sap_publiccloud.pm

```diff
@@ -1043,12 +1065,13 @@ sub get_hana_site_names {
 
 =head2 create_hana_vars_section
 
-    Detects HANA/HA scenario from openQA variables and creates
+    Detects HANA/HA scenario from openQA variables and create
     data later used for "ansible: hana_vars:" section in config.yaml file.
 
 =cut
 
 sub create_hana_vars_section {
+    my ($ha_enabled) = @_;
```

Comment:
I think it is a mistake in rebasing https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/21245/files#diff-4b7291e584e1d0b487f685a7b45efc74dcd804ee2236acf7a866003d0f4fb87aL1051

This argument is mo more needed